babel/src/test/resources/sql/spark.iq

```diff
@@ -240,4 +240,108 @@ EXPR$0
 true
 !ok
 
+#####################################################################
+# CONCAT_WS
+#
+# CONCAT_WS(sep[, str | array(str)]+)
+# Returns the concatenation of the strings separated by sep, skipping null values.
+# If sep is null, returns null directly
+#
+# Returns STRING
+
+SELECT CONCAT_WS('s');
+EXPR$0
+
+!ok
+
+select CONCAT_WS(',', null);
+EXPR$0
+
+!ok
+
+SELECT CONCAT_WS(',', null, null);
+EXPR$0
+
+!ok
+
+SELECT CONCAT_WS(',', null, 'a');
+EXPR$0
+a
+!ok
+
+SELECT CONCAT_WS(',', 'a', 'b');
+EXPR$0
+a,b
+!ok
+
+SELECT CONCAT_WS(',', 'a', null, 'b');
+EXPR$0
+a,b
+!ok
+
+SELECT CONCAT_WS(null, 'a', 'b');
+EXPR$0
+null
+!ok
+
+SELECT CONCAT_WS(',', 'a', 100, 'b');
+EXPR$0
+a,100,b
+!ok
+
+SELECT CONCAT_WS(',', 'a', 100.0, 'b');
+EXPR$0
+a,100.0,b
+!ok
+
+select CONCAT_WS('', cast('a' as varchar(2)), cast('b' as varchar(1)));
+EXPR$0
+ab
+!ok
+
+select CONCAT_WS(',', array['a', 'b', 'c']);
+EXPR$0
+a,b,c
+!ok
+
+select CONCAT_WS(',', array['a', null, 'c']);
+EXPR$0
+a,c
+!ok
+
+select CONCAT_WS(',', array['a']);
+EXPR$0
+a
+!ok
+
+select CONCAT_WS(null, array['a']);
+EXPR$0
+null
+!ok
+
+select CONCAT_WS(null, array['a'], array['b'], array['c']);
+EXPR$0
+null
+!ok
+
+select CONCAT_WS(',', array['a'], array['b'], array['c']);
```

Comment:
In Spark. This SQL should be :
`select concat_ws(',', array('a'), array('b'), array('c'));`

Reply:
Updated the test cases.